frontend/src/views/c-authorship.vue

```diff
@@ -803,20 +803,21 @@ export default {
       }
 
       .mui-form--inline {
+        @include small-font;
         align-items: flex-end;
         display: flex;
         flex-wrap: wrap;
 
         #search {
-          @include medium-font;
+          @include small-font;
```

Comment:
Can I ask about the reason for changing the font size here?

Reply:
The toolbar controls on the summary panel were using a smaller font whereas the ones on the authorship panel were using `medium-font`. To unify the design for these controls that have similar purposes, I have defined a `small-font` with a size between the initial 2 sizes.